### PR TITLE
Will retry message to appear before 60 seconds wait time

### PR DESCRIPTION
**Pull request by Shrankhlas**
Customer doesn't wait for 60 seconds for retrying to happen if not prompted with a retry message. These code changes are implemented so that a message appears before retry actually happens.

### CODE REVIEW

**Comment by Shrankhlas** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: push_physical_badging_records_ITARL5.ps1 [hunk 1]
```diff
@@ -11,7 +11,9 @@ param
     [Parameter(mandatory = $true)]
     [string] $jobId,
     [Parameter(mandatory = $true)]
-    [string] $jsonFilePath
+    [string] $jsonFilePath,
+    [Parameter(mandatory = $false)]
+    [Int] $retryTimeout
```
Comment (sasanche) [inline push_physical_badging_records_ITARL5.ps1:16]: same as above

**Comment by sasanche** [comment]
> Customer doesn't wait for 60 seconds for retrying to happen if not prompted with a retry message. These code changes are implemented so that a message appears before retry actually happens.

@vrajsoniMS , @Shrankhlas  Added a few comments in PR, PR is already merged though, without any approvals, are we not following code review process for this repo?

**Comment by vrajsoniMS** [comment]
The rules were not configured on this repo, added rules to get an approval before merging.
Missed the default value, @Shrankhlas please create a followup PR to update

**Comment by Shrankhlas** [comment]
https://github.com/microsoft/m365-physical-badging-connector-sample-scripts/pull/8 
Here is the pr with updates

**Pull request merged**